homeassistant/components/aurora_abb_powerone/strings.json

```
@@ -16,7 +16,7 @@
     },
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
-      "no_serial_ports": "No com ports found.  Need a valid RS485 device to communicate."
+      "no_serial_ports": "No com ports found. Need a valid RS485 device to communicate."
```

```suggestion
      "no_serial_ports": "No com ports found. The integration needs a valid RS485 device to communicate."
```
The sentence sounded like caveman talk